antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,386 @@
+                        {% else %}
+                          Removed in: a future release
+                        {% endif %}
+                        {% if value['deprecated']['removed_from_collection'] and value['deprecated']['removed_from_collection'] != collection %}
+                          of @{ value['deprecated']['removed_from_collection'] | escape }@
+                        {% endif %}
+                        <br>
+                        Why: @{ value['deprecated']['why'] | escape }@
+                        <br>
+                        Alternative: @{ value['deprecated']['alternative'] | escape }@
+                      </div>
+                    {% endif %}
+                </td>
+                {# default / choices #}
+                <td>
+                    {# Turn boolean values in 'yes' and 'no' values #}
+                    {% if value['default'] is sameas true %}
+                        {% set _x = value.update({'default': 'yes'}) %}
+                    {% elif value['default'] is not none and value['default'] is sameas false %}
+                        {% set _x = value.update({'default': 'no'}) %}
+                    {% endif %}
+                    {% if value['type'] == 'bool' %}
+                        {% set _x = value.update({'choices': ['no', 'yes']}) %}
+                    {% endif %}
+                    {# Show possible choices and highlight details #}
+                    {% if value['choices'] %}
+                        <ul style="margin: 0; padding: 0"><b>Choices:</b>
+                            {% for choice in value['choices'] %}
+                                {# Turn boolean values in 'yes' and 'no' values #}
+                                {% if choice is sameas true %}
+                                    {% set choice = 'yes' %}
+                                {% elif choice is sameas false %}
+                                    {% set choice = 'no' %}
+                                {% endif %}
+                                {% if (value['default'] is not list and value['default'] == choice) or (value['default'] is list and choice in value['default']) %}
+                                    <li><div style="color: blue"><b>@{ choice | escape }@</b>&nbsp;&larr;</div></li>
+                                {% else %}
+                                    <li>@{ choice | escape }@</li>
+                                {% endif %}
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                    {# Show default value, when multiple choice or no choices #}
+                    {% if value['default'] is not none and value['default'] not in value['choices'] %}
+                        <b>Default:</b><br/><div style="color: blue">@{ value['default'] | tojson | escape }@</div>
+                    {% endif %}
+                </td>
+                {# configuration #}
+                {% if plugin_type != 'module' %}
```

Comment:
I suppose either this part of the template should be removed, or the conditional should be updated to detect roles (and skip), since they wouldn't be able to use the configuration column, right?

Reply:
Indeed. I removed it in 6fbbe1d01c72ebb6694f5fe97676266f1dabe619 and updated my docsite with the latest version.